sunpy/net/fido_factory.py

```diff
@@ -385,10 +385,23 @@ def fetch(self, *query_results, path=None, max_conn=5, progress=True,
         if "wait" in kwargs:
             raise ValueError("wait is not a valid keyword argument to Fido.fetch.")
 
+        # TODO: Hack for JSOC to avoid more than one connection.
+        # Remove when parfive allows us to special case URLS.
+        is_jsoc_only = False
```

Comment:
Does this mean a query that returns non-JSOC and JSOC results will still use more than one connection to download the JSOC results?

Reply:
yes.

Reply:
I added more of a comment here.